server/protocols/jmap/src/main/java/org/apache/james/jmap/methods/SetMessagesCreationProcessor.java

```diff
@@ -114,7 +113,7 @@ private void handleCreate(CreationMessageEntry create, Builder responseBuilder,
         try {
             validateImplementedFeature(create, mailboxSession);
             validateArguments(create, mailboxSession);
-            validateRights(create, mailboxSession);
+            validateIsUserOwnerOfMailboxes(create, mailboxSession);
```

Comment:
assertUserIsOwnerOfMailboxes (we should keep validate when we expect validation results)